src/Photos/Enums.cs

```diff
@@ -378,12 +382,35 @@ public enum FigExifCustomRenderedValue : short {
 	[TV (13,0), Mac (10,15), iOS (13,0)]
 	[Native]
 	public enum PHPhotosError : long {
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use 'InternalError' instead.")]
 		Invalid = -1,
+		InternalError = -1,
 		UserCancelled = 3072,
 		LibraryVolumeOffline = 3114,
 		RelinquishingLibraryBundleToWriter = 3142,
 		SwitchingSystemPhotoLibrary = 3143,
+		[TV (14,0), Mac (11,0), iOS (14,0)]
```

Comment:
It's a special case but we do not annotate error codes with OS versions. 

Why ? They are are output-only and, as enum values, they do not have to exists to be used. E.g.

```csharp
var error = CallApi ();
switch (error)
case PHPhotosError. NetworkAccessRequired:
    // it does not make sense to add a version check here
    // and we don't want compiler/tooling warnings for them if we don't 
    Console.WriteLine ("Network Access Required");
    break;
// ... more cases ...
default:
    Console.WriteLine ($"Error {error}");
    break;
}
```

It's also a bunch of (not really useful) metadata to include inside platform assemblies.

Reply:
that makes sense, thank you! removing now